sample/db/samples/reimbursements.rb

```diff
@@ -2,21 +2,43 @@
 
 Spree::Sample.load_sample("orders")
 
-order          = Spree::Order.last
-inventory_unit = order.inventory_units.first
+order = Spree::Order.last
+inventory_unit = order.inventory_units.take!
 stock_location = inventory_unit.find_stock_item.stock_location
+return_reason = Spree::ReturnReason.active.take!
+preferred_reimbursement_type = Spree::ReimbursementType.where(name: 'Original').take!
```

Comment:
As far as I remember the `take` method expects the collection to contain at least one element, this will probably break if we don't have any `Spree::ReimbursementType` in the database